src/components/application_manager/include/application_manager/application_manager_impl.h

```diff
@@ -1416,7 +1417,7 @@ class ApplicationManagerImpl
   // Thread that pumps messages audio pass thru to mobile.
   impl::AudioPassThruQueue audio_pass_thru_messages_;
 
-  HMICapabilities hmi_capabilities_;
+  std::auto_ptr<HMICapabilities> hmi_capabilities_;
```

Comment:
@LevchenkoS Concerning this class field I have no objections - reviewed. The question was asked in order to trigger a discussion in an appropriate moment.